package.json

```diff
@@ -38,13 +40,15 @@
     "shelljs": "0.8.5",
     "shx": "0.3.4",
     "snyk": "^1.364.0",
-    "ts-loader": "5.4.5",
+    "ts-jest": "28.0.8",
+    "ts-loader": "^9.3.0",
+    "ts-node": "10.9.1",
     "tslint": "5.5.0",
-    "typescript": "3.8.3",
+    "typescript": "^4.7.4",
     "vsce": "1.103.1",
     "vscode-test": "1.6.1",
-    "webpack": "4.30.0",
-    "webpack-cli": "3.3.12"
+    "webpack": "^5.74.0",
+    "webpack-cli": "^4.10.0"
```

Comment:
Note this versions should be consistent across all the packages.

Reply:
Have verified these version bumps across the packages. it's consistent.